scripts/initializers/order.js

```diff
@@ -23,6 +23,16 @@ await initializeDropin(async () => {
   const orderRef = searchParams.get('orderRef');
   const returnRef = searchParams.get('returnRef');
   const isTokenProvided = orderRef && orderRef.length > 20;
+  const labels = await fetchPlaceholders();
+  const langDefinitions = {
+    default: {
+      ...labels,
+    },
+  };
+
+  if (pathname.includes(CUSTOMER_ORDERS_PATH)) {
```

Comment:
I might have better idea 🤔 

```js
  // Handle redirects for user details pages
  if (pathname === ORDER_DETAILS_PATH
    || pathname === CUSTOMER_ORDER_DETAILS_PATH
    || pathname === RETURN_DETAILS_PATH
    || pathname === CUSTOMER_RETURN_DETAILS_PATH
    || pathname === CREATE_RETURN_PATH
    || pathname === CUSTOMER_CREATE_RETURN_PATH) {
    console.log('INITIALIZED WITH REDIRECTS');
    await handleUserOrdersRedirects(
      pathname,
      isAccountPage,
      orderRef,
      returnRef,
      isTokenProvided,
      langDefinitions,
    );

    return;
  }

  console.log('INITIALIZED WITHOUT REDIRECTS');
  await initializers.mountImmediately(initialize, {
    langDefinitions,
    orderRef,
    returnRef,
  });
```

We can go this way. Initialise with redirects on specific pages and initialise without redirects on any other page.

But you will still need to add two new URLs as constants into first check.

Reply:
Updated